### ISSUE TITLE
Vagrant 1.9.7, Parallels Desktop 12 name of VM in box changed by "vagrant up"

### ISSUE THREAD

**Issue by janssen** [Parallels/vagrant-parallels#303]
I haven't been able to create a VM with Vagrant 1.9.7 and Parallels Desktop 12 on MacOS Sierra.  I'm building my own base box.  When I then "vagrant init" from that box, it seems to work OK.  But when I then try "vagrant up" on that new machine, the act seems to change the pathname of the stored VM in the Vagrant box, and "up" fails.  Here's a log:

```
% vagrant box list
�[0;33mThere are no installed boxes! Use `vagrant box add` to add some.�[0m
% vagrant box add --name=test /tmp/custom.box
�[1m==> box: Box file was not detected as metadata. Adding it directly...�[0m
�[1m==> box: Adding box 'test' (v0) for provider: �[0m
�[0m    box: Unpacking necessary files from: file:///tmp/custom.box�[0m
�[K�[0m�[1;32m==> box: Successfully added box 'test' (v0) for 'parallels'!�[0m
% ls -F ~/.vagrant.d/boxes/test/0/parallels
metadata.json	tester.pvm/
% ls -F ~/.vagrant.d/boxes/test/0/parallels/tester.pvm/
Snapshots.xml				VmInfo.pvi				parallels.log
Ubuntu Linux 16.04 Desktop-0.hdd/	config.pvs*				statistic.log
Ubuntu Linux 16.04 Desktop.app/		config.pvs.backup*			vm.lock
% vagrant init test
�[0mA `Vagrantfile` has been placed in this directory. You are now
ready to `vagrant up` your first virtual environment! Please read
the comments in the Vagrantfile as well as documentation on
`vagrantup.com` for more information on using Vagrant.�[0m
% ls -F ~/.vagrant.d/boxes/test/0/parallels
metadata.json	tester.pvm/
% ls -F ~/.vagrant.d/boxes/test/0/parallels/tester.pvm/
Snapshots.xml				VmInfo.pvi				parallels.log
Ubuntu Linux 16.04 Desktop-0.hdd/	config.pvs*				statistic.log
Ubuntu Linux 16.04 Desktop.app/		config.pvs.backup*			vm.lock
% vagrant up
�[0mBringing machine 'default' up with 'parallels' provider...�[0m
�[1m==> default: Registering VM image from the base box 'test'...�[0m
/Users/wjanssen/.vagrant.d/gems/2.3.4/gems/vagrant-parallels-1.7.5/lib/vagrant-parallels/action/box_register.rb:54:in `initialize': No such file or directory @ rb_sysopen - /Users/wjanssen/.vagrant.d/boxes/test/0/parallels/tester.pvm/config.pvs (Errno::ENOENT)
	from /Users/wjanssen/.vagrant.d/gems/2.3.4/gems/vagrant-parallels-1.7.5/lib/vagrant-parallels/action/box_register.rb:54:in `open'
	from /Users/wjanssen/.vagrant.d/gems/2.3.4/gems/vagrant-parallels-1.7.5/lib/vagrant-parallels/action/box_register.rb:54:in `box_id'
	from /Users/wjanssen/.vagrant.d/gems/2.3.4/gems/vagrant-parallels-1.7.5/lib/vagrant-parallels/action/box_register.rb:113:in `register_box'
	from /Users/wjanssen/.vagrant.d/gems/2.3.4/gems/vagrant-parallels-1.7.5/lib/vagrant-parallels/action/box_register.rb:27:in `block (2 levels) in call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/environment.rb:631:in `lock'
	from /Users/wjanssen/.vagrant.d/gems/2.3.4/gems/vagrant-parallels-1.7.5/lib/vagrant-parallels/action/box_register.rb:26:in `block in call'
	from /Users/wjanssen/.vagrant.d/gems/2.3.4/gems/vagrant-parallels-1.7.5/lib/vagrant-parallels/action/box_register.rb:24:in `synchronize'
	from /Users/wjanssen/.vagrant.d/gems/2.3.4/gems/vagrant-parallels-1.7.5/lib/vagrant-parallels/action/box_register.rb:24:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:34:in `call'
	from /Users/wjanssen/.vagrant.d/gems/2.3.4/gems/vagrant-parallels-1.7.5/lib/vagrant-parallels/action/customize.rb:39:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:34:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:34:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/builder.rb:116:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/runner.rb:66:in `block in run'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/util/busy.rb:19:in `busy'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/runner.rb:66:in `run'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/builtin/call.rb:53:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:34:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/builtin/config_validate.rb:25:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:34:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:34:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/builtin/handle_box.rb:56:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:34:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:95:in `block in finalize_action'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:34:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/builder.rb:116:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/runner.rb:66:in `block in run'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/util/busy.rb:19:in `busy'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/runner.rb:66:in `run'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/builtin/call.rb:53:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/warden.rb:34:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/builder.rb:116:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/runner.rb:66:in `block in run'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/util/busy.rb:19:in `busy'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/action/runner.rb:66:in `run'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/machine.rb:227:in `action_raw'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/machine.rb:202:in `block in action'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/environment.rb:631:in `lock'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/machine.rb:188:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/machine.rb:188:in `action'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.7/lib/vagrant/batch_action.rb:82:in `block (2 levels) in run'
% ls -F ~/.vagrant.d/boxes/test/0/parallels
box_lease_count		metadata.json		tester (3).pvm/
%
```

**Comment by legal90** [comment]
Hi @janssen,
The output you've provided shows that `tester.pvm` has been renamed to `tester (3).pvm` right after the box registration. Parallels Desktop does that in the scope of automatic VM rename, which happens if there is a VM with the same name already registered. 

I thin we can work it around in vagrant-parallels by re-checking the path to `*.pvm` right before trying to read the `config.pvs` file inside it.

**Comment by legal90** [comment]
BTW, subsequent runs of `vagrant up` should work fine.

**Comment by legal90** [comment]
Fixed in `vagrant-parallels` v1.7.6  relese (https://github.com/Parallels/vagrant-parallels/releases/tag/v1.7.6)
